CONTRIBUTING.md

```diff
@@ -11,33 +11,29 @@ Submitting code to the project? Please review and follow our
 
 ### Code Quality
 
-Although we do not have official code style guidelines, we can and will request you to make changes
-if we think that your code is sloppy. You can take clues from the existing code base to see what we
-consider to be reasonable code quality. Please be prepared to make changes that we ask of you even
-if you might not agree with the request(s).
+Although we do not have official code style guidelines, we can and will request you to make changes if we think that your code is sloppy. You can take clues from the existing code base to see what we consider to be reasonable code quality. Please be prepared to make changes that we ask of you even if you might not agree with the request(s).
```

Comment:
I do not think we should use the word "sloppy" here.

Reply:
It was being used before

Reply:
That doesn't mean it was "nice" to begin with.